CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Fixed
 - Grouping of text lines outside of parent container bounding box ([#386](https://github.com/pdfminer/pdfminer.six/pull/386))
 
+### Changed
```

Comment:
I think this is added functionality. It depends on how you define "functionality" though. As I see it, all the documented input/output remain the same, we just *add* a new option to pass `None`.